cunumeric/eager.py

```diff
@@ -485,21 +486,49 @@ def swapaxes(self, axis1: int, axis2: int) -> NumPyThunk:
         self.children.append(result)
         return result
 
-    def convert(self, rhs: Any, warn: bool = True) -> None:
+    def convert(
+        self,
+        rhs: Any,
+        warn: bool = True,
+        nan_op: Optional[int] = None,
```

Comment:
you can make a similar change here to simplify the code.

Reply:
bd361b6dc3a341df6a95f957f32321f70827ad38